inc/functions-shortcode.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * Shortcode
+ */
+
+function latin_activity_shortcode( $atts ) {
+
+	$a = shortcode_atts( array(
+		'beginner-activity' => '',
+		'advanced-activity' => ''
+	), $atts );
+
+	switch ( $a['beginner-activity'] ) {
+		case '1':
+			return embed_code( 'four-three', '01', 'beginner-activities', 'activity01.swf' );
+			break;
+		case 'p2':
+			return embed_code( 'four-three', 'p02', 'beginner-activities', 'practice02.swf' );
+			break;
+		case '2':
+			return embed_code( 'four-three', '02', 'beginner-activities', 'activity02.swf' );
+			break;
+		case 'p3':
+			return embed_code( 'four-three', 'p03', 'beginner-activities', 'practice03.swf' );
+			break;
+		case '3':
+			return iframe_activity( '125%', '/beginner-activities/activity03.html' );
```

Comment:
Flash files using iframes are not rendering at full width in Firefox and Chrome. They are full width in Safari however.

Reply:
I think it's now fixed. CSS wasn't targeting embed code inside the iframe, so I added the CSS directly into the HTML doc.